ddr/ddr/Models/GameEntities/Rider.swift

```diff
@@ -30,24 +28,24 @@ class Rider: SKSpriteNode {
     }
   }
   
-  var lives:Int = 3 {
-    didSet {
-      if (lives == 0) {
-        loseGame()
-      } else {
-        respawn()
-      }
-    }
-  }
+  var lives:Int = 3
+//  {
+//    didSet {
+//      if (lives == 0) {
+//        isDead = true
+////        loseGame()
+//      } else {
+////        respawn()
```

Comment:
commenting this line out means that there's no period of invincibility after getting hit. Is this intended?

Reply:
It was. No flashing and no invincibility, and I'm cool with it. Resolved!